src/components/Comments.jsx

```diff
@@ -123,6 +148,22 @@ export default function Comments({ article_id }) {
                                 <p className="title">
                                     <strong>{comment.author}</strong>,{" "}
                                     {elapsedTimeString}
+                                    {loggedInUser?.username ===
+                                    comment.author ? (
+                                        <Link
+                                            className="delete"
+                                            onClick={(event) => {
+                                                // DEAR TUTOR, IF YOU SEE THIS!!!
+                                                // I couldn't think of another way to disable my delete button, in the event that there is some lag from the server
+                                                event.target.onclick = () => {};
+                                                handleDelete(
+                                                    comment.comment_id
+                                                );
+                                            }}
+                                        >
```

Comment:
Aha so you can't disable this because you are using Link which is a react component from react router. If you change it to be a button then you will be able to disable